YaseminAydin.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html>
+	<head>
+		<title> This is Me </title>
+		 <a> https://www.linkedin.com/in/yasemin-aydin-1414959b/ </a>
```

Comment:
This doesn't create a link for your LinkedIn page.
You should add the url to the `href` attribute like below. The text in between the `<a></a>` tags will show up on the page.

```html
<a href="https://www.linkedin.com/in/yasemin-aydin-1414959b/">LinkedIn</a>
```

See here for more info: [https://www.w3schools.com/tags/att_a_href.asp](https://www.w3schools.com/tags/att_a_href.asp)